### PR TITLE
 lang update urdu 

### PR DESCRIPTION
**Pull request by Xisdom**

### CODE REVIEW

**Comment by Xisdom** [comment]
Urdu translation

**Comment by Xisdom** [comment]
Xisdom:src_config_lang_Urdu_translation

File: web/src/config/lang/ur.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
 		"TEXT": "بلا جھجھک {0} مزید معلومات اور کسی بھی مسئلے کے لیے ہمیں ای میل بھیجیں۔"
 	},
 	"HELP_TELEGRAM_TEXT": "اوپن اے پی ا ی دستاویزات دیکھیں:",
-	"HELP_TELEGRAM_LINK": "https://apidocs.hollaex.com",
+	"HELP_TELEGRAM_LINK": "ایچ ٹی ٹی پی ایس ://اےپی ای ڈوکس۔ہولاایکس ۔کام",
```
Comment (Xisdom) [inline web/src/config/lang/ur.json]: has been corrected

**Comment by abeikverdi** [comment]
Please open the PR ro hotfix branch instead

**Pull request closed without merging**